inspirehep/modules/workflows/errors.py

```diff
@@ -115,3 +115,15 @@ def __init__(self, workflow_id, **kwargs):
         super(CallbackWorkflowNotInMergeState, self).__init__(**kwargs)
         self.message = 'Workflow {} is not in merge state.'.format(
             workflow_id)
+
+
+class CallbackWfNotInWaitingForCurationState(CallbackError):
```

Comment:
Seeing that we are generalizing the workflow for any edit, would make more sense to rename this to `CallbackWorkflowNotInWaitingEditState` or similar.

Reply:
yep, now everything changes accordingly!